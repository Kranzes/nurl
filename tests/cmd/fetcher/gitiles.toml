args = ["https://android.googlesource.com/platform/external/minijail", "linux-v18"]
