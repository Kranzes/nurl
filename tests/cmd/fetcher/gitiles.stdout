fetchFromGitiles {
  url = "https://android.googlesource.com/platform/external/minijail";
  rev = "linux-v18";
  hash = "sha256-OpwzISZ5iZNQvJAX7UJJ4gELEaVfcQgY9cqMM0YvBzc=";
}